package.json

```diff
@@ -28,21 +29,63 @@
   "homepage": "https://github.com/octokit/app.js#readme",
   "types": "index.d.ts",
   "dependencies": {
-    "@octokit/request": "^3.0.0",
-    "@types/lru-cache": "^5.1.0",
+    "@octokit/request": "^4.0.1",
     "jsonwebtoken": "^8.3.0",
     "lru-cache": "^5.1.1"
   },
   "devDependencies": {
-    "chai": "^4.1.2",
+    "@gr2m/pika-plugin-bundle-web": "^0.3.16-patch-issue-46-6",
+    "@pika/pack": "^0.3.7",
+    "@pika/plugin-build-node": "^0.3.16",
+    "@pika/plugin-build-web": "^0.3.16",
+    "@pika/plugin-ts-standard-pkg": "^0.3.16",
+    "@types/jest": "^24.0.13",
+    "@types/jsonwebtoken": "^8.3.0",
+    "@types/lolex": "^3.1.1",
+    "@types/lru-cache": "^5.1.0",
+    "@types/nock": "^10.0.3",
+    "@types/simple-mock": "^0.8.1",
     "coveralls": "^3.0.2",
+    "jest": "^24.8.0",
     "lolex": "^4.0.1",
-    "mocha": "^6.1.1",
     "nock": "^10.0.2",
-    "nyc": "^14.0.0",
+    "prettier": "^1.17.1",
     "semantic-release": "^15.12.1",
     "simple-mock": "^0.8.0",
```

Comment:
I think simple-mock is no longer used?

Reply:
It's still used, though the test that it's used in doesn't work anymore